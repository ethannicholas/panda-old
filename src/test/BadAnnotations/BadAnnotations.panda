@class
@thread
@external
@override
@math(overflow)
@limited
@self
@safeReturn
class Foo {
}