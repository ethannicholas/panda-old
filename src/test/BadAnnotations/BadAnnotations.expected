error: BadAnnotations.panda:1:1: @self may not be used together with @class
error: BadAnnotations.panda:1:1: @override may not be used together with @class
error: BadAnnotations.panda:1:1: @thread may not be used together with @class
error: BadAnnotations.panda:1:1: @limited may not be used together with @self
error: BadAnnotations.panda:9:1: annotation '@class' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@thread' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@external' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@override' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@math(overflow)' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@limited' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@self' is not supported on classes
error: BadAnnotations.panda:9:1: annotation '@safeReturn' is not supported on classes
12 errors
